app/javascript/Pages/SingleClass.jsx

```diff
@@ -0,0 +1,220 @@
+  };
+
+  return (
+    <Layout current_user={props.current_user}>
+      <div className="path-wrapper">
+        <Link rel="stylesheet" href="/home" className="path">
+          <img
+            src={require("../images/logo_dark.svg")}
+            alt="logo"
+            className="path-logo"
+          />
+        </Link>
+        <p className="path">&nbsp;/&nbsp;</p>
+        <Link rel="stylesheet" href={classes_url_category()} className="path">
+          {props.class.skill.category.name}
+        </Link>
+        <p className="path">&nbsp;/&nbsp;</p>
+        <Link rel="stylesheet" href={classes_url_skill()} className="path">
+          {props.class.skill.name}
+        </Link>
+        <p className="path">&nbsp;/&nbsp;</p>
+        <Link rel="stylesheet" href="" className="path">
+          {props.class.teacher.name}
+        </Link>
+      </div>
+      <div
+        className="class-header"
+        style={{ "--colorCategory": `#${props.class.skill.category.color}` }}
+      >
+        <h1 className="class-header-title">{props.class.title}</h1>
+        <h2 className="class-header-subtitle">{props.class.teacher.name}</h2>
+      </div>
+      <div>
+        <ul className="class-menu">
+          <li
+            className={click == "about" ? "class-btn active" : "class-btn"}
+            onClick={() => handleClick("about")}
+            style={{
+              "--colorCategory": `#${props.class.skill.category.color}`,
+            }}
+          >
+            About
+          </li>
+          <li
+            className={click == "review" ? "class-btn active" : "class-btn"}
+            onClick={() => handleClick("review")}
+            style={{
+              "--colorCategory": `#${props.class.skill.category.color}`,
+            }}
+          >
+            Review
+          </li>
+          <li
+            className={click == "comments" ? "class-btn active" : "class-btn"}
+            onClick={() => handleClick("comments")}
+            style={{
+              "--colorCategory": `#${props.class.skill.category.color}`,
+            }}
+          >
+            Comments
+          </li>
+        </ul>
+      </div>
+
+      <div className={click == "about" ? "class-about" : "class-about off"}>
+        <div className="class-details">
+          <p className="class-difficulty">{props.class.difficulty}</p>
+          <div className="class-lessons">
+            <h1 className="class-details-title">{props.class.no_classes}</h1>
+            <p className="class-details-subtitle">Lessons </p>
+          </div>
+          <div className="class-time">
+            <h1 className="class-details-title">
+              {props.class.class_duration}
+            </h1>
+            <p className="class-details-subtitle">Time p/lesson</p>
+          </div>
+        </div>
+        <div className="class-details-sub">
+          {props.class.regime}
+          &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
It's a funny line, but if want space between elements you shouldn't try to achieve that by adding a dozen of unicodes for a white space.